modules/php/States/PlayCardTrait.php

```diff
@@ -66,6 +87,14 @@ public function arg_playCard()
     if ($playRule == -1) {
       return ["count" => "All but 1"];
     }
+
+    $addInflation = Utils::getActiveInflation() ? 1 : 0;
+    $partyBonus =
+      Utils::getActivePartyBonus() && Utils::isPartyInPlay()
+        ? 1 + $addInflation
+        : 0;
+    $playRule += $addInflation + $partyBonus;
```

Comment:
I believe there is an issue here with "play all but 1". We need to make sure $playRule != 1 before adding the inflation, partyBonus & richBonus.